components/common/MainNav.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Link } from 'routes';
+import TetherComponent from 'react-tether';
+
+// components
+import SubMenu from 'components/common/SubMenu';
+
+// helpers
+import { pushTo } from 'utils/router';
```

Comment:
I don't really understand the need for this helper because you still have to write one line.